products/.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we will have more products?
If not I suggest to move it into the `releng` folder. The target and repository project can then reside there as well (when we move to a deeper project structure).

Reply:
> Do we will have more products?

don't know, but I'd like to follow structured layout here, a product has nothing to do with releng (=release engineering) stuff.

Reply:
I wouldn't say nothing, but yes products if is of course the most suitable name.
However, would the .project file then better be in the folder containing the .product instead of the container?

Reply:
As a product file do not need a project at all I don't think we need (dedicated) projects per product here.

Reply:
OK. I both is valid, but since this your change, you decide. :)

I just wonder in general if Tycho then builds multiple products in the same project as long as they are in separate directories? IIRC having multiple products in the same directory does not work (well).

Reply:
Tycho (pomless) builds each folder, but using multiple in one folder would also work...